specs/trust-store-trust-policy.md

```diff
@@ -81,9 +81,31 @@ Users who consume signed artifacts from OCI registries, or signed arbitrary blob
+  - **`signatureVerification`**(*object*): This REQUIRED property dictates how signature verification is performed.
+  An *object* that specifies a predefined verification level, with an option to override the Notary Project trust policy defined verification level if user wants to specify a [custom verification level](#custom-verification-level).
+    - **`level`**(*string*): A REQUIRED property that specifies the verification level, supported values are `strict`, `permissive`, `audit` and `skip`. Detailed explanation of each level is present [here](#signatureverification-details).
+    - **`override`**(*map of string-string*): This OPTIONAL map is used to specify a [custom verification level](#custom-verification-level).
+  - **`timestampVerification`**(*object*): This OPTIONAL property dictates how timestamp countersignature verification is performed. If present, it is used solely under the [`notary.x509`](./signing-scheme.md/#notaryx509) signing scheme. If not present, default timestamp countersignature verification will be performed. Current supported fields are:
+    - **`level`**(*string*): A REQUIRED property that specifies the verification level, supported values are `strict` and `skip`. `strict` is the default value meaning timestamp countersignature verification is performed. `skip` means no timestamp countersignature verification is performed.
+    - **`expiryRelaxed`**(*boolean*): OPTIONAL. When set to `true`, validate the timestamp certificate chain expiration against the time point been stamped. When not set or set to `false`, validate the timestamp certificate chain expiration against the time at verification.
+  - **`trustStores`**(*map of trust-store-type -> array-of-named-stores*): REQUIRED map with `trust-store-type` as keys and corresponding `array of named trust stores` as values. Each named trust store contains the trusted root certificates against which signatures are verified. Currently supported `trust-store-type` are `ca`, `signingAuthority` and `tsa`. For example, when TSAs are specified, the format is `"tsa":[acme-tsa, wabbit-tsa]`.
+  - **`trustedIdentities`**(*map of trust-store-type -> array-of-trusted-identities*): REQUIRED map with `trust-store-type` as keys and corresponding `array of trusted identities` as values. Currently supported `trust-store-type` are `ca`, `signingAuthority` and `tsa`. For example, when a CA trusted identity is specified, the format is `"ca": ["x509.subject: C=US, ST=WA, L=Seattle, O=acme-rockets.io, OU=Finance, CN=SecureBuilder"]`. A value `*` is supported if user trusts any identity under a certain `trust-store-type`. For example, `"tsa": ["*"]`. A missing `trust-store-type` or empty array in the map means NO identity is trusted under the particular trust store type.
+
+    For X.509 PKI, it supports list of elements/attributes of the signing certificate's subject. For more information, see [identities constraints](#trusted-identities-constraints) section. 
+  
+
 #### Blob Trust Policy
```

Comment:
Is it possible to unify the trust policies for both OCI artifacts and blobs?

Reply:
I might miss your question here. Do you mean merge `blob trust policy` and `oci trust policy` back to ONE trust policy? 
I don't think it's doable at this point, because they use very different mechanism to pick applicable trust policy statements during verification.